src/main/java/org/sagebionetworks/web/client/view/ProfileViewImpl.java

```diff
@@ -378,7 +381,8 @@ public void showTabs(boolean isOwner) {
 		if (isOwner) {
 			resetHighlightBoxes();
 			DisplayUtils.show(settingsListItem);
-			DisplayUtils.show(downloadsListItem);
+			if (!DisplayUtils.isInTestWebsite(cookies))
```

Comment:
Do not show the Download tab in the Profile Dashboard if in alpha mode (it's on it's own page now).